base.tf

```diff
@@ -0,0 +1,32 @@
+provider "aws" {
+  access_key = var.aws_access_key
+  secret_key = var.aws_secret_key
+  region     = var.aws_region
+}
+
+
+resource "aws_s3_bucket" "b" {
+  bucket = "dannybucket99sss9"
+  acl    = "private"
+
+  tags = {
+    Name        = "danny bucket"
+    Environment = "Dev"
+  }
+}
+
+resource "aws_s3_bucket_object" "test1" {
+  bucket  = "dannybucket99sss9"
```

Comment:
Could you check how can you add this as reference of the aws_s3_bucket resource? This is going to avoid the use of "depends_on" because it's going to be an implicit reference.

Reply:
Sure! I've just added the implicit reference